packages/berry-fslib/sources/CwdFS.ts

```diff
@@ -1,219 +1,36 @@
-import {posix}                                             from 'path';
-
-import {CreateReadStreamOptions, CreateWriteStreamOptions} from './FakeFS';
-import {FakeFS, WriteFileOptions}                          from './FakeFS';
-import {NodeFS}                                            from './NodeFS';
+import {FakeFS}              from './FakeFS';
+import {NodeFS}              from './NodeFS';
+import {ProxiedFS}           from './ProxiedFS';
+import {PortablePath, ppath} from './path';
 
 export type CwdFSOptions = {
-  baseFs?: FakeFS,
+  baseFs?: FakeFS<PortablePath>,
 };
 
-export class CwdFS extends FakeFS {
-  private readonly target: string;
+// FIXME: notable changes:
+// - symlink didn't transform the target path, it does now
```

Comment:
I'm pretty sure this was a bug and this comment can be ignored, but I added it so I didn't forget to validate that assumption

Reply:
That sounds about right (especially since we call `fromPortablePath` on it in `NodeFS`) - if the tests pass I'd say we can keep this change.